regression/esbmc-cpp11/constructors/Constructor1-3/main.cpp

```diff
@@ -5,13 +5,13 @@ class t2
 public:
   int i;
 
-  t2() : i(2)
+  t2() : i(1)
   {
   }
 };
 
 int main()
 {
   t2 instance2;
-  assert(instance2.i == 2);
+  assert(instance2.i == 1);
 }
```

Comment:
Maybe this test case doesn't need to be changed?

Reply:
ACK